### ISSUE TITLE
macOS safari the background color of selected text didn't respect to system accent color

### ISSUE THREAD

**Issue by kuige** [darkreader/darkreader#1875]
When changing system accent color in macOS , the background of selected text in safari is still blue, not the accent color.

**Comment by Gusted** [comment]
We have no way of accesing the system accent color in macOS it isn't exposed to Extension.

**Comment by kuige** [comment]
@Gusted When dark reader is activated, this css code snippet is added to the page by dark reader, leading to the background of selected text always be blue.

<img width="301" alt="image" src="https://user-images.githubusercontent.com/8212800/87213777-fea5b600-c359-11ea-8cf8-eb2e5fe1c08c.png">

I have verified that, if this code snippet was removed, the background color of selected text will respect to the system accent color.

When dark reader turns off, there is no styles for the selected text added to the page, so the background color of selected text automatically respect to the system accent color.

So I think the solution of this issue is just remove the code snippet above.
